_sass/_team.scss

```diff
@@ -1,10 +1,10 @@
-.profile-img{
-    display: block;
-    max-width: 70%;
-    height: auto;
-    margin-left: auto;
-    border-radius: 4px;
-    @media (max-width: 700px) {
-        margin-right: auto;
-    }
+.profile-img {
+    width: 30%;
+    display: flex;
+    justify-content: flex-start;
+    border-radius: 25px;
```

Comment:
Is there any reason you changed the `border-radius` here? Now it looks unsymmetric.